### PR TITLE
adding the view on unilective on the context menu

### LINKED ISSUE #1132: Add a "view course on unilectives" button in the context menu
For when you right click on a course in term planner.

### PR DESCRIPTION
**Pull request by jacksonwanggg**
image

### CODE REVIEW

File: frontend/src/pages/TermPlanner/ContextMenu/ContextMenu.tsx [hunk 1]
```diff
@@ -58,6 +59,11 @@ const ContextMenu = ({ code, plannedFor, ignoreFromProgression }: Props) => {
     ignoreFromProgressionMutation.mutate(code);
   };
 
+  const handleUnilective = () => {
+    window.open(`https://unilectives.devsoc.app/course/${code}`, '_blank');
+    dispatch(addTab(code));
```
Comment (ollibowers) [inline frontend/src/pages/TermPlanner/ContextMenu/ContextMenu.tsx]: Im not sure we want to `dispatch(addTab(code))` here. 

This adds the course to the course selector page tabs.

**Review by ollibowers — CHANGES_REQUESTED** [review]
just one small fix, otherwise looks good to me!

File: frontend/src/pages/TermPlanner/ContextMenu/ContextMenu.tsx (hunk 1 above)
Comment (jacksonwanggg) [reply frontend/src/pages/TermPlanner/ContextMenu/ContextMenu.tsx]: ohh right i just saw every url navigation having it so i just also added sorry ;(

**Review by ollibowers — APPROVED** [review]
looks great to me!

**Pull request merged**